internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -325,7 +325,7 @@ func addPGMonitorExporterToInstancePodSpec(
 		}, {
 			Name: "exporter-queries",
 			// this is the path for the default config, which we generate and then mount via ConfigMap
-			MountPath: pgmonitor.QueriesConfigDir,
+			MountPath: "/tmp",
```

Comment:
Is there any overlap in the `tmp` directory with other containers? Is that an issue?